pyrokinetics/pyroscan.py

```diff
@@ -395,7 +402,9 @@ def set_in_dict(data_dict, map_list, value):
     """
     Sets item in dict given location as a list of string
     """
-    get_from_dict(data_dict, map_list[:-1])[map_list[-1]] = value
+    from copy import deepcopy, copy
```

Comment:
This one is complaining about `copy.copy` being imported and not used

```suggestion
    from copy import deepcopy
```